conans/client/command.py

```diff
@@ -1332,12 +1354,44 @@ def alias(self, *args):
 
         self._conan.export_alias(args.reference, args.target)
 
+    def link(self, *args):
+        """ Creates a link from a reference to a local package.
```

Comment:
It would be more clear something like: `Links a conan reference (e.g lib/1.0@conan/stable) with a local folder path`